docs/examples/Track2Track_Fusion_Example.py

```diff
@@ -0,0 +1,784 @@
+gmlcc_tracks.update(gmlcc_tracker.tracks)
+meas_fusion_tracks.update(meas_fusion_tracker.tracks)
+track_fusion_tracks.update(track_fusion_tracker.tracks)
+
+# Remove tracks that have just one state in them as they were probably from clutter
+jpda_tracks = set([track for track in jpda_tracks if len(track) > 1])
+gmlcc_tracks = set([track for track in gmlcc_tracks if len(track) > 1])
+meas_fusion_tracks = set([track for track in meas_fusion_tracks if len(track) > 1])
+track_fusion_tracks = set([track for track in track_fusion_tracks if len(track) > 1])
+
+# Add tracks to metric managers
+sensor1_mm.add_data(tracks=jpda_tracks, overwrite=False)
+sensor2_mm.add_data(tracks=gmlcc_tracks, overwrite=False)
+meas_fusion_mm.add_data(tracks=meas_fusion_tracks, overwrite=False)
+track_fusion_mm.add_data(tracks=track_fusion_tracks, overwrite=False)
+
+# %%
+# 9: Plot the Results
+# --------------------
+# Next, we will plot all of the resulting tracks and measurements. This will be done in two plots.
+# The first plot will show all of the data, and the second plot will show a closer view of one
+# resultant track.
+#
+# These plots are done in 2D to make them more readable. We invite the reader to explore the plot
+# interactively using the following line in an active Jupyter session.
+#
+# %matplotlib widget
+
+# %%
+plotter1, plotter2 = Plotter(), Plotter()
+for plotter in [plotter1, plotter2]:
+    plotter.plot_ground_truths(set(radar_simulator.groundtruth.groundtruth_paths), [0, 2],
+                               color='black')
+    plotter.plot_measurements(sensor1_detections, [0, 2], color='orange', marker='*',
+                              measurements_label='Measurements - Airborne Radar')
+    plotter.plot_measurements(sensor2_detections, [0, 2], color='blue', marker='*',
+                              measurements_label='Measurements - Ground Radar')
+    plotter.plot_tracks(jpda_tracks, [0, 2], color='red',
+                        track_label='Tracks - Airborne Radar (JPDAF)')
+    plotter.plot_tracks(gmlcc_tracks, [0, 2], color='purple',
+                        track_label='Tracks - Ground Radar (GM-LCC)')
+    plotter.plot_tracks(meas_fusion_tracks, [0, 2], color='green',
+                        track_label='Tracks - Measurement Fusion (GM-PHD)')
+    plotter.plot_tracks(track_fusion_tracks, [0, 2], color='pink',
+                        track_label='Tracks - Covariance Intersection (GM-PHD)')
+
+    # Format the legend a bit. Set the position outside of the plot, and
+    # swap the order of the clutter and ground radar measurements
+    pos = plotter.ax.get_position()
+    plotter.ax.set_position([pos.x0, pos.y0, pos.width * 0.7, pos.height])
+    k = list(plotter.legend_dict.keys())
+    k[2], k[3] = k[3], k[2]
+    v = list(plotter.legend_dict.values())
+    v[2], v[3] = v[3], v[2]
+    plotter.ax.legend(handles=v, labels=k, loc='lower center', bbox_to_anchor=(0.5, -0.5))
+
+plotter1.fig.show()
+
+track = track_fusion_tracks.pop()
```

Comment:
Building on CircleCI currently fails as `track_fusion_tracks` is an empty set.

Reply:
Ah I see, thanks Steve! I added an if statement to check if the track_fusion_tracks has any tracks in it before plotting. I've run the notebook so many times on my own, and it very rarely happens that there are no tracks. So I think this is an adequate solution.